pkg/katautils/config.go

```diff
@@ -130,6 +130,8 @@ type hypervisor struct {
 	HotplugVFIOOnRootBus    bool     `toml:"hotplug_vfio_on_root_bus"`
 	DisableVhostNet         bool     `toml:"disable_vhost_net"`
 	GuestHookPath           string   `toml:"guest_hook_path"`
+	RxRateLimiter           int64    `toml:"rx_rate_limiter"`
```

Comment:
ooi, since zero means "default", why can't these be `uint64`'s? In fact, looking at FC, everything to do with rate limiting is defined in terms of `u64`:

- https://github.com/firecracker-microvm/firecracker/blob/master/src/rate_limiter/src/persist.rs#L14

Reply:
thanks!!! you're right. ;)